config/routes.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+Rails.application.routes.draw do
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
+  # root "articles#index"
+  namespace :api do
+    namespace :v1 do
+      resources :messages, only: [:index, :show, :create, :update, :destroy]
```

Comment:
### Optional
- [ ] It would be nice if we expose only the needed routes, since our controller only make use of the `index` method. Let us kindly delete the others

Reply:
Thanks man. Would fix that